models/app.models.js

```diff
@@ -72,55 +64,47 @@ exports.postNewComment = (article_id, commentToPost) => {
                     VALUES ($1, $2, $3) RETURNING author as username, body`;
   const queryValue = [];
   const queryProms = [];
-  if (article_id && commentToPost) {
-    queryValue.push(body, article_id, author);
-    queryProms.push(checkExists("articles", "article_id", article_id));
-  }
+
+  queryValue.push(body, article_id, author);
+  queryProms.push(checkExists("articles", "article_id", article_id));
   queryProms.push(db.query(queryString, queryValue));
+
   return Promise.all(queryProms).then((promResults) => {
-    
-    if (queryProms.length === 1) {
-      return promResults[0].rows[0];
-    } else {
-      return promResults[1].rows[0];
-    }
+    return promResults[1].rows[0];
   });
 };
 
 exports.selectArticleToPatch = (article_id, inc_votes) => {
-  const queryString = `UPDATE articles SET votes = votes + $1 WHERE article_id = $2 RETURNING*`
-  const queryValue = [inc_votes]
-  const queryProms = []
-  if (article_id){
-    queryValue.push(article_id)
-    queryProms.push(checkExists("articles", "article_id", article_id))
-  }
-  queryProms.push(db.query(queryString, queryValue))
+  const queryString = `UPDATE articles SET votes = votes + $1 WHERE article_id = $2 RETURNING*`;
+  const queryValue = [inc_votes];
+  const queryProms = [];
+
+  queryValue.push(article_id);
+  queryProms.push(checkExists("articles", "article_id", article_id));
+  queryProms.push(db.query(queryString, queryValue));
+
+  return Promise.all(queryProms).then((promResults) => {
+    return promResults[1].rows[0];
+  });
+};
+
+exports.selectCommentToDelete = (comment_id) => {
+  let queryString = ``;
+  const queryValue = [];
+  const queryProms = [];
+
+  queryString += `DELETE FROM comments WHERE comment_id = $1 RETURNING*`;
+  queryValue.push(comment_id);
+  queryProms.push(checkExists("comments", "comment_id", comment_id));
```

Comment:
No need to check that this exists - if psql can't find the comment to delete it will throw an error